samples/jenkins/scripts/prod/run_model_deployment_validation.sh

```diff
@@ -0,0 +1,37 @@
+export PATH=$PATH:$PWD
+
+results_path = 'prod_job_results.zip'
+
+find_asset () {
+  echo "Searching for $1 with name: $2..." >&2
+  asset_id=$(cpdctl asset search --type-name $1 --query "asset.name:$2" \
+    --output json --jmes-query "results[0].metadata.asset_id" --raw-output)
+  echo "Found: $asset_id" >&2
+  echo $asset_id
+}
+
+prod_space_id=$(cpdctl space list --output json --jmes-query "resources[?entity.name == 'prod-space'] | [0].metadata.id" --raw-output)
+export CPD_SCOPE=cpd://cpd/spaces/$prod_space_id
+
+prod_evaluation_script_id=$(find_asset script "evaluate_model.py")
+
+evaluate_model_job_id=$(cpdctl job list --output json --jmes-query "results[?metadata.name == 'evaluate_model_job'] | [0].metadata.asset_id")
+echo "Run starting for a job: $evaluate_model_job_id..."
+
+cpdctl job run create --job-id "$evaluate_model_job_id" --job-run '{}'
+
+results_asset_id=$(cpdctl asset search --type-name data_asset --query "evaluation_result.zip*" --output json \
+  --jmes-query "results[0].metadata.asset_id" --raw-output)
+
+echo "Results : $result_asset_id ..."
+
+results_attachment_id=$(cpdctl asset get --asset-id "$results_asset_id" --output json --jmes-query "attachments[-1].id" --raw-output)
+
+echo "Downloading: $evaluation_output_name to the $results_path..."
+
+cpdctl asset attachment download  --asset-id "$results_asset_id" --attachment-id "$results_attachment_id" --output-path "$results_path"
+
+echo "Unziping $results_path to results.txt"
+
+unzip -p "$results_path" results.txt
```

Comment:
Would it be possible to print also the result of the evaluation in the script?

Reply:
There is no way to produce log from the script run (if this is what you ask for)